Gemfile

```diff
@@ -32,7 +32,7 @@ gem "uglifier", ">= 1.3.0", require: false
 # Explicitly avoid 1.x that doesn't support Ruby 2.4+
 gem "json", ">= 2.0.0"
 
-gem "rubocop", ">= 0.47", require: false
+gem "rubocop", ">= 0.53", require: false
```

Comment:
To make RuboCop version the same between local environment and Code Climate, it also need to change the channel of codeclimate.yml.
https://github.com/rails/rails/blob/master/.codeclimate.yml#L26

However, Code Climate doesn't seem to be ready to make the channel 0.53 yet.
https://github.com/codeclimate/codeclimate-rubocop/issues/118

AFAIK, you will have to wait for that release to synchronize versions.

Reply:
@koic Thanks for pointing that out. I just realised that earlier today too. A possible interim solution would be to have a separate .rubocop.yml for codeclimate using channel 0.52. But the codeclimate team seems fairly quick with adding new channels so we can just simply wait for them.